README.md

```diff
@@ -116,6 +116,14 @@ nexusIQScan {
 - Open Terminal on the project's root and run `./gradlew nexusIQScan`
 - You should see the scan report URL report on Terminal.
 
+### Nexus IQ Index
+Allow to save information about the dependencies of a project into a module information file (module.xml) that Sonatype CI tools can use to include these dependencies in a scan.
+```
+nexusIQIndex {
+     modulesExcluded = ['module-1', 'module-2'] // Optional. For multi-module projects, the names of the sub-modules to exclude from indexing.
```

Comment:
Out of curiosity is this meant to be the equivalent of how maven can exclude modules during a build? e.g.

```
mvn com.sonatype.clm:clm-maven-plugin:index -pl "!clm-maven-plugin-its"
```

will exclude the `clm-maven-plugin-its` module from being indexed

Reply:
> Out of curiosity is this meant to be the equivalent of how maven can exclude modules during a build? e.g.
> 
> ```
> mvn com.sonatype.clm:clm-maven-plugin:index -pl "!clm-maven-plugin-its"
> ```
> 
> will exclude the `clm-maven-plugin-its` module from being indexed

Indeed. I'm using `modulesExcluded` because it's a name already used in another existing task.